.github/workflows/release.yml

```
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: 'Build and upload Go binaries'
+    runs-on: ubuntu-latest
+
+    steps:
+      # Install packages
+      - name: Install packages
+        shell: bash
+        run: |
+          apt-get update
+          apt-get install --yes ca-certificates curl jq
+
+      # Setup Golang
+      - name: Setup Golang
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.1'
+
+      # Checkout the branch (into `runatlantis` path since this fork uses `runatlantis` Go imports)
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          path: ./src/github.com/runatlantis/atlantis
+
+      # Build binaries
+      - name: Build binaries
+        shell: bash
+        # https://github.com/mvdan/github-actions-golang#how-do-i-set-up-a-gopath-build
+        # https://help.github.com/en/articles/virtual-environments-for-github-actions
+        # https://laszlo.cloud/setting-gopath-in-github-actions
+        # https://laszlo.cloud/setting-gopath-in-github-actions-revisited
+        env:
+          GOPATH: ${{ runner.workspace }}
+          GOBIN: $GOPATH/bin
+          PATH: $PATH:$GOBIN
+          GO111MODULE: off
+          CGO_ENABLED: 0
+        run: |
+          .github/workflows/build-binaries.sh release
+          ls -l release/
+
+      # Upload binaries
+      - name: Upload binaries
+        shell: bash
+        run: |
+          .github/workflows/upload-binaries.sh release
```

Seems like it expects a list of files.

```suggestion
          .github/workflows/upload-binaries.sh release/*
```